README.md

```diff
@@ -18,9 +18,17 @@ In this milestone, I implemented the following;
 - Implemented the CategoryFilter component
 - Implemented processes to keep the filter value in the redux store.
 
-## Preview
+### Milestone 5
+In this milestone, I have implemented the UI design of the project using the specific design provided for it. At this time, everything about the frontend design and feature implementation of this project is complete. 
+
+## Main Features
+- User can add a new book
+- User can remove a book
+- User can filter books based on category.
 
-## Features
+## Preview
+![image](images/bookstore-1.png)
+![image](images/bookstore.png)
 
 ## Project SetUp
 Get a local copy of the file. Clone with HTTPS following the steps below.
```

Comment:
## Fix the UI layout on large screens
- [ ] For this milestone, we need to ensure nothing is messed up. I believe we are using pixels to size our elements while styling our components on the DOM using our development screens. But we need to slack off a bit on using fixed values when trying to style components for different screen sizes. Do you mind improving on this? Thank you!

|[Design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)|[Implementation](https://zubenna-bookstore.herokuapp.com/)|
|:--:|:--:|
|![source](https://user-images.githubusercontent.com/22345961/114283384-22fe0280-9a52-11eb-898d-c3b44bc9845a.png)|![clone](https://user-images.githubusercontent.com/22345961/114283395-33ae7880-9a52-11eb-8db5-7da1fc309297.png)|